lib/filewatch/winhelper.rb

```diff
@@ -192,7 +188,11 @@ def self.char_pointer_to_ruby_string(char_pointer, length = 256)
   end
 
   def self.utf16le(string)
-    string.encode("UTF-16LE")
+    to_cstring(string).encode("UTF-16LE")
+  end
+
+  def to_cstring(rubystring)
```

Comment:
Unless this module extends it-`self`, when calling the module method `Winhelper::utf16le(String)`, this instance method will not be available.

```suggestion
  def self.to_cstring(rubystring)
```

Reply:
damn, you are correct. Thanks.